target_chains/ethereum/contracts/contracts/pyth/PythAccumulator.sol

```diff
@@ -170,6 +185,19 @@ abstract contract PythAccumulator is PythGetters, PythSetters, AbstractPyth {
         }
     }
 
+    function parseWormholeMerkleHeaderNumUpdates(
```

Comment:
IMO it's better to reuse the method above instead of adding this. I don't think getUpdateFee needs to be efficient now, since it's only used by off-chain callers.

Reply:
ah yes good point about that on-chain usage.